operator-common/src/main/java/io/strimzi/operator/common/model/Labels.java

```diff
@@ -110,6 +110,13 @@ public static Labels fromResource(HasMetadata resource) {
         return new Labels(resource.getMetadata().getLabels() != null ? resource.getMetadata().getLabels() : emptyMap());
     }
 
+    /**
+     * Returns the labels from Map.
+     */
+    public static Labels fromMap(Map<String, String> labels) {
```

Comment:
Not for this PR, but I do look at `Labels` and wonder how helpful it is now that we have fewer Strimzi-specific labels. It's not really encapsulating anything: It's just a Map, really. We could achieve what we need with a few helper functions.

Reply:
I think the Labels class and its interface are useful. But maybe once we have some time we can do an analysis of what of the methods is actually still needed because I guess some might be already obsolete etc.